<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWidget</class>
 <widget class="QWidget" name="ExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size: 16pt;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Export data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="expandDetailsCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
	color: #CA0B5E;
	padding: 0.5em;
	font-size: 13pt;
	outline: none;
}
QWidget:focus {
	border: 0.1em solid #CA0B5E;
}
QWidget:hover {
	background-color: rgba(202,11,94,0.8);
	color: white;
}
QWidget:disabled {
	color: grey;
	border-color: grey;
}
</string>
       </property>
       <property name="text">
        <string>Expand Details</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
	border: 0.05em solid #CA0B5E;
	border-radius: 0.25em;
	color: #CA0B5E;
	padding: 0.5em;
	font-size: 13pt;
	outline: none;
}
QWidget:focus {
	border: 0.1em solid #CA0B5E;
}
QWidget:hover {
	background-color: rgba(202,11,94,0.8);
	color: white;
}
QWidget:disabled {
	color: grey;
	border-color: grey;
}
</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
